models/src/main/scala/coop/rchain/casper/PrettyPrinter.scala

```diff
@@ -38,12 +39,14 @@ object PrettyPrinter {
     }
 
   def buildString(d: ProcessedDeploy): String =
-    s"User: ${buildStringNoLimit(d.deploy.deployer)}, Cost: ${d.cost.toString} " +
+    s"User: ${buildStringNoLimit(d.deploy.pk.bytes)}, Cost: ${d.cost.toString} " +
```

Comment:
you changed the output of the `show-block` command which is one of reason fail the integration tests. I suggest you add the sig in the end instead of replacing the original content.

Reply:
I don't think that output should be changed because of this. Now are strict rules for signed deploys so maybe now deployer can be different (correct).
Can you post an example? I see signature printed on the end in this function, I'm not sure that I understand what you mean.